pkg/scheduler/framework/session.go

```diff
@@ -315,9 +317,9 @@ func updateRootQueueResources(ssn *Session, allocated v1.ResourceList) {
 		return
 	}
 
-	if !equality.Semantic.DeepEqual(queue.Spec.Deserved, totalResource) ||
+	if !equality.Semantic.DeepEqual(queue.Spec.Deserved, totalDeserved) ||
 		!equality.Semantic.DeepEqual(queue.Spec.Guarantee.Resource, totalGuarantee) {
-		queue.Spec.Deserved = totalResource
+		queue.Spec.Deserved = totalDeserved
```

Comment:
The queue here is a new queue and does not specify the capability, if the user sets the capability for the root queue, it needs to be set here as well